### ISSUE TITLE
Allow modifying how {{authorString}} is parsed

### ISSUE THREAD

**Issue by HEmile** [hans/obsidian-citation-plugin#41]
I link to all authors on a paper in my graph, ie [[author1]], [[author2]], [[author3]]...
It'd be nice if it were possible to modify how `{{authorString}}` is parsed to allow automatically inserting these links.

**Comment by hans** [comment]
Hey @HEmile , good idea! Implemented in #42, and it'll be part of a near-future release.

**Comment by psyguy** [comment]
I was just about to open an issue for this feature. Great minds indeed think alike, @HEmile. 😄

(And thanks a lot, @hans, for this much-needed plugin.)

**Comment by hans** [comment]
#42 is merged and this will be released in 0.4.0.

**Comment by jeffkimbrel** [comment]
This is exactly what I was looking for, and I got the authors to print in a list format that is yaml compatible with 

```
authors: [{{#each entry.author}}"{{this.family}}, {{this.given}}"{{#unless @last}}, {{/unless}}{{/each}}]
```

Thanks so much!
